File: CS/MvpvmNavigation/Interfaces.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DevExpress.XtraBars.Ribbon;

namespace MvpvmNavigation {
    public interface IRibbonModule {
        RibbonControl Ribbon { get; }
    }

    public interface ISupportTransitions {
        void StartTransition(bool forward, object waitParameter);
        void EndTransition();
    }
}

File: VB/MvpvmNavigation/Interfaces.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports DevExpress.XtraBars.Ribbon

Namespace MvpvmNavigation
    Public Interface IRibbonModule
        ReadOnly Property Ribbon() As RibbonControl
    End Interface

    Public Interface ISupportTransitions
        Sub StartTransition(ByVal forward As Boolean, ByVal waitParameter As Object)
        Sub EndTransition()
    End Interface
End Namespace
